jvb/src/main/java/org/jitsi/videobridge/cc/BitrateController.java

```diff
@@ -417,6 +419,7 @@ public JSONObject getDebugState()
         debugState.put("lastN", lastN);
         debugState.put("supportsRtx", supportsRtx);
         debugState.put("oversending", oversending);
+        debugState.put("total_oversending_time_secs", oversendingTimeTracker.getTotalOversendingTime().getSeconds());
```

Comment:
I don't see that level of precision being necessary here.

Reply:
I guess `update()` usually runs infrequently enough that we're not worried about total oversending times of < 1 second?

Reply:
I guess this is the only case where we'd maybe care about the precision, as it would look like it had never happened when, in fact, it had for some short amount of time.  Generally though I think the level of seconds would be fine as we're just not doing anything very precise with this number.

Reply:
Ok.  It's an easy enough change if we want to change it later.